### ISSUE TITLE
Crazy-paving pattern

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6691]
**Preferred term label:**
Crazy-paving pattern

**Definition (free text, please give PubMed ID)**
This pattern appears as thickened interlobular septa and intralobular lines superimposed on a background of ground-glass opacity, resembling irregularly shaped paving stones. The crazy-paving pattern is often sharply demarcated from more
normal lung and may have a geographic outline. It was originally reported in patients with alveolar proteinosis and is also encountered in other diffuse lung diseases that affect both the interstitial and airspace compartments, such as lipoid pneumonia.
PMID:18195376

Comment
See Figure 20 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
